test/celeste.test.ts

```diff
@@ -12,7 +12,7 @@ const TORUS_TEST_EMAIL = "hello@tor.us";
 const TORUS_TEST_VERIFIER = "torus-test-health";
 const TORUS_TEST_AGGREGATE_VERIFIER = "torus-test-health-aggregate";
 
-describe("torus utils celeste", function () {
+describe.skip("torus utils celeste", function () {
```

Comment:
sorry, should have commented on this in my first review:
is this `skip` intentional?

otherwise, looks good.